net-misc/sipp/sipp-3.6.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="SIPp is a free Open Source test tool / traffic generator for the SIP protocol"
+HOMEPAGE="http://sipp.sourceforge.net/ https://github.com/SIPp/sipp/releases"
+SRC_URI="https://github.com/SIPp/sipp/releases/download/v${PV}/${PF}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl +pcap sctp +ssl"
+
+DEPEND="sys-libs/ncurses:=
+	ssl? ( dev-libs/openssl:= )
+	pcap? ( net-libs/libpcap net-libs/libnet:1.1 )
+	gsl? ( sci-libs/gsl:= )
+"
+RDEPEND="${DEPEND}"
+
+src_configure()
```

Comment:
Just follow the default style and place `{` here. I know some people don't like the syntax, but let's not be different.

Reply:
Sorry, I generally write C/C++ code with the { on the next line (reads easier IMHO).  Fixed locally.